### PR TITLE
CHANGELOG entry for version 1.12.0

### PR DESCRIPTION
**Pull request by jlstevens**
- [ ] Get feedback and make appropriate fixes/changes.
- [ ] Update releases.rst

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,58 @@
+Version 1.12.0
+==============
+
+This release provides a number of exciting new features as well as a set
+of important bug fixes. Many thanks to our users for reporting these
+issues and @poplarShift @reckoner @jonmease for their code contributions
+and @scottclowe @syhooper for their contributions to the documentation.
```
Comment (philippjfr) [inline CHANGELOG.md]: All the ands and the lack of commas made this a bit ugly so maybe just merge them together:

>Many thanks to our users for reporting these issues, and special thanks to @poplarShift, @reckoner, @jonmease, @scottclowe and @syhooper for their contributions to the code and the documentation.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,58 @@
...
+
+Features:
+
+- New plot options for controlling layouts including a responsive mode
+  as well as improved control over aspect using the newly updated bokeh
+  layout engine ([#3450](https://github.com/pyviz/holoviews/pull/#3450),
+  [#3575](https://github.com/pyviz/holoviews/pull/#3450))
+
```
Comment (philippjfr) [inline CHANGELOG.md]: Should match formatting of previous entries which was without spaces between entries.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,59 @@
...
+- Other miscellaneous fixes
+([#3530](https://github.com/pyviz/holoviews/pull/#3450),
+[#3536](https://github.com/pyviz/holoviews/pull/#3450),
+[#3546](https://github.com/pyviz/holoviews/pull/#3450),
+[#3560](https://github.com/pyviz/holoviews/pull/#3450),
+[#3571](https://github.com/pyviz/holoviews/pull/#3450),
+[#3580](https://github.com/pyviz/holoviews/pull/#3450),
+[#3584](https://github.com/pyviz/holoviews/pull/#3450),
+[#3585](https://github.com/pyviz/holoviews/pull/#3450),
+[#3594](https://github.com/pyviz/holoviews/pull/#3450))
```
Comment (philippjfr) [inline CHANGELOG.md]: All the links are pointing to the same PR.

Comment (philippjfr) [reply CHANGELOG.md]: And they're not valid URLs.

File: CHANGELOG.md (hunk 1 above)
Comment (jlstevens) [reply CHANGELOG.md]: Fixed in df58ab544

File: CHANGELOG.md (hunk 3 above)
Comment (jlstevens) [reply CHANGELOG.md]: Urgh thanks for spotting that. My macro went wrong..

Comment (jlstevens) [reply CHANGELOG.md]: Fixed in c562f6d

File: CHANGELOG.md (hunk 2 above)
Comment (jlstevens) [reply CHANGELOG.md]: Fixed in 107e313

**Pull request merged**